test/data/recordings/TestAccBrandingTheme.yaml

```diff
@@ -1,290 +1,290 @@
-        null
-      form: { }
-      headers:
-        Content-Type:
-          - application/json
-        User-Agent:
-          - Go-Auth0-SDK/latest
-      url: https://terraform-provider-auth0-dev.eu.auth0.com/api/v2/branding/themes/w1BL2SDwg4oWU4F83iTVzESMzbmGFgf8
-      method: GET
-    response:
-      proto: HTTP/2.0
-      proto_major: 2
-      proto_minor: 0
-      transfer_encoding: [ ]
-      trailer: { }
-      content_length: -1
-      uncompressed: true
-      body: '{"displayName":"My branding","borders":{"button_border_radius":2,"button_border_weight":2,"buttons_style":"pill","input_border_radius":2,"input_border_weight":2,"inputs_style":"pill","show_widget_shadow":true,"widget_border_weight":2,"widget_corner_radius":2},"colors":{"base_focus_color":"#BBBBBB","base_hover_color":"#CCCCCC","body_text":"#00FF00","error":"#00FF00","header":"#00FF00","icons":"#00FF00","input_background":"#00FF00","input_border":"#00FF00","input_filled_text":"#00FF00","input_labels_placeholders":"#00FF00","links_focused_components":"#00FF00","primary_button":"#00FF00","primary_button_label":"#00FF00","secondary_button_border":"#00FF00","secondary_button_label":"#00FF00","success":"#00FF00","widget_background":"#00FF00","widget_border":"#00FF00"},"fonts":{"body_text":{"bold":true,"size":100},"buttons_text":{"bold":true,"size":100},"font_url":"https://google.com/font.woff","input_labels":{"bold":true,"size":100},"links":{"bold":true,"size":100},"links_style":"normal","reference_text_size":12,"subtitle":{"bold":true,"size":100},"title":{"bold":true,"size":100}},"page_background":{"background_color":"#000000","background_image_url":"https://google.com/background.png","page_layout":"center"},"widget":{"header_text_alignment":"center","logo_height":55,"logo_position":"center","logo_url":"https://google.com/logo.png","social_buttons_layout":"top"},"themeId":"w1BL2SDwg4oWU4F83iTVzESMzbmGFgf8"}'
-      headers:
-        Content-Type:
-          - application/json; charset=utf-8
-      status: 200 OK
-      code: 200
-      duration: 1ms
-  - id: 7
-    request:
-      proto: HTTP/1.1
-      proto_major: 1
-      proto_minor: 1
-      content_length: 0
-      transfer_encoding: [ ]
-      trailer: { }
-      host: terraform-provider-auth0-dev.eu.auth0.com
-      remote_addr: ""
-      request_uri: ""
-      body: ""
-      form: { }
-      headers:
-        Content-Type:
-          - application/json
-        User-Agent:
-          - Go-Auth0-SDK/latest
-      url: https://terraform-provider-auth0-dev.eu.auth0.com/api/v2/branding/themes/w1BL2SDwg4oWU4F83iTVzESMzbmGFgf8
-      method: DELETE
-    response:
-      proto: HTTP/2.0
-      proto_major: 2
-      proto_minor: 0
-      transfer_encoding: [ ]
-      trailer: { }
-      content_length: 0
-      uncompressed: false
-      body: ""
-      headers:
-        Content-Type:
-          - application/json; charset=utf-8
-      status: 204 No Content
-      code: 204
-      duration: 1ms
+    - id: 0
```

Comment:
Regarding your testing question, we have probably have another step that doesn't set any of the options and then checks that the defaults get applied. e.g.

```
const testAccBrandingThemeCreate = `
resource auth0_branding_theme my_theme {

}
`

// ... and then add another test step to check defaults in the actual test func
```

Reply:
Hi! @sergiught I've made the changes as requested and tried to get it to a point where none of the options are set and the defaults are applied. There may be a neater way of doing it, more than happy for you to push changes on this branch. Thanks!